### ISSUE TITLE
Dark theme support for Gradle syntax highlighting

### ISSUE THREAD

**Issue by vogella** [eclipse/buildship#682]
The current color for the Gradle syntax highlighting in the dark theme makes the text unreadable.

![selection_303](https://user-images.githubusercontent.com/139910/38361695-87fd14ca-38ce-11e8-87a3-4605f0e41caf.png)

Please fix, if you need help how to do this, please let me know.

**Comment by vogella** [comment]
Looks like this is only a theme switch issue. After closing and re-opening the editor it looks better.

![selection_305](https://user-images.githubusercontent.com/139910/38362775-58c2c520-38d2-11e8-9180-fafee63546e2.png)


Maybe you can add a preference listener for the color font switch and refresh the editor?

**Comment by donat** [comment]
That's correct, the editor only inherits the color scheme from the Java editor, but no preference listening is added. The fix sounds relatively easy. @vogella if you can provide a pr, that would be awesome ;)

**Comment by vogella** [comment]
It is at the moment unlikely that I find time to contribute a fix, my
current backlog in platform is already larger than my available time.

Donát Csikós <notifications@github.com> schrieb am Fr., 6. Apr. 2018, 10:11:

**Comment by JacquesLeRoux** [comment]
Hi,

Dark theme now suggest a change (via a button) in the Egradle editor preferences that works well with darkest theme.
